core/templates/pages/preferences-page/form-fields/preferred-languages.component.ts

```diff
@@ -41,8 +41,15 @@ export class PreferredLanguagesComponent {
   removable = true;
   separatorKeysCodes: number[] = [ENTER];
   formCtrl = new FormControl();
+  filteredChoices: LanguageIdAndText[] = [];
+  searchQuery: string = '';
 
-  ngOnInit(): void {
+  onInputBoxClick(): void {
```

Comment:
Does this function reset search state? In that case, it might be make more sense to rename it to something like `resetLanguageSearch` and call it again below (line 85)

Reply:
Done